content/plugins/fabric.md

```diff
@@ -31,14 +31,19 @@ As the fabric plugin is used for remote execution, the fact that it doesn't supp
 {{% /gsNote %}}
 
 
-## Execution Methods
+# Execution Methods
 
 There are 4 modes for working with this plugin.
 
 * Executing a list of `commands`.
 * Executing a Fabric task from a `tasks_file` included in the blueprint's directory.
 * Executing a Fabric task by specifying its path in the current python environment.
-* Executing a script by specifying the script's path or URL.
+* Executing a `script` by specifying the script's path or URL.
+
+## Executing commands or scripts with sudo
+
+Previously, to run `commands` or commands within a `script` using sudo, you had to explicitly write `sudo` where desired. Starting with fabric-plugin v1.4.1, The `commands` and `script` methods both accept a `use_sudo` input (which defaults to `false`). When true, the commands or script will be executed using sudo. This allows, for instance, to use the `sudo_prefix` fabric env property to run an alternative implementation of sudo. See below for examples on how to pass `use_sudo` and use the `sudo_prefix` property.
```

Comment:
I'm not sure this is the place to mention that this wasn't available previously.
Simply explain the feature